core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -126,11 +135,12 @@
   }
 
   /**
-   * Returns the SslSocketFactory singleton, which can be used to create SslSockets to Cloud SQL.
+   * Returns the CoreSocketFactory singleton, which can be used to createSslSocket SslSockets to
```

Comment:
```suggestion
   * Returns the {@link CoreSocketFactory} singleton.
```

What it does is documented elsewhere.